packages/mdl-radio/index.js

```diff
@@ -0,0 +1,43 @@
+import MDLBaseComponent, {
+  MDLBaseAdapter
+} from 'mdl-base-component';
+import MDLRadioMixin, {
+  Identifier
+} from './mixin';
+import EventBus from './event-bus';
+
+const DEFAULT_EVENT_BUS = new EventBus();
+
+export default class MDLRadio extends MDLBaseComponent {
+  static attachTo(root, eventBus = DEFAULT_EVENT_BUS) {
+    const nativeInput = root.getElementsByClassName(Identifier.NATIVE_INPUT)[0];
```

Comment:
Since most components (including the base component) just accept a `root` parameter with one node representing the component, it may be cleaner to do the same here and perform this step within the constructor.